### DISCUSSION TITLE
[dapps/W3MWagmi] [react-native] [hardhat] How to connect a wallet on iOS simulator or how to connect with a Hardhat?

### DISCUSSION THREAD

**Discussion by damdafayton** [WalletConnect/walletconnect-monorepo#4053] [Developer Support]
Hi all,
I did setup [this](https://github.com/WalletConnect/react-native-examples/tree/main/dapps/W3MWagmi) react-native app to study it a bit and then integrate into my own app. But I have no clue about how to test it locally because when I click on 'connect wallet' ios simulator warns that 'App is not installed'.

What is the most proper way of connecting a wallet to a dapp through this SDK locally for testing?
Is there a way to auto connect with a Hardhat address or a burner wallet to bypass this connection?
Im new in dapp develeopment. Ive been searcing over net since few days to how to setup this but couldnt find a resource unfortunately.

**Comment by ignaciosantise** [comment]
Hi @damdafayton 👋 iOS emulators don't allow to install apps. To test locally you could:

* Switch to an Android Emulator and install some wallets from PlayStore
* Connect to a Browser Wallet, like Zerion
* Use a physical device to test

**Reply by damdafayton** [reply]
i know ios simulator doesnt allow that, thats why ive asked this question.

i want to connect to the Hardhat network preferably with a burner wallet automatically for local testing.

i dont want to use physical device at this point.

ill give a try to your suggestion about browser wallet and see if it makes things easier.

thanks.
